anaconda_project/test/test_project.py

```diff
@@ -2777,3 +2777,11 @@ def check(dirname):
 
     with_directory_contents_completing_project_file(
         {DEFAULT_PROJECT_FILENAME: "env_specs:\n  foo:\n    packages: [something]\n    somejunk: True\n"}, check)
+
+
+def test_empty_file_has_problems():
+    def check(dirname):
+        project = project_no_dedicated_env(dirname)
+        assert ['anaconda-project.yml: The env_specs section is empty.'] == project.problems
```

Comment:
This in general is a bit brittle if we decide to reword things no?

Maybe creating actual Problem objects or Exceptions of Warning would make testing easier, independent on the wording of the error message?

(just a suggestion for future iterations)